README.md

```diff
@@ -1 +1,78 @@
-# TMBAnalysis
+# tmbAnalysis
+
+workflow to calculate TMB
+
+## Overview
+
+## Dependencies
+
+
+
+## Usage
+
+### Cromwell
+```
+java -jar cromwell.jar run tmbAnalysis.wdl --inputs inputs.json
+```
+
+### Inputs
+
+#### Required workflow parameters:
+Parameter|Value|Description
+---|---|---
+`inputMaf`|File|input maf file
+`intervalFile`|String|target bed file
+`outputFileNamePrefix`|String|Prefix to use for output file
+
+
+#### Optional workflow parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+
+
+#### Optional task parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+`calculateTMB.modules`|String|"tmb-r/1.0"|module for running preprocessing
+`calculateTMB.jobMemory`|Int|4|memory allocated to preprocessing, in GB
+`calculateTMB.timeout`|Int|6|timeout in hours
+
+
+### Outputs
+
+Output | Type | Description
+---|---|---
+`outputTMB`|File|output TMB file
+
+
+## Commands
+ This section lists command(s) run by WORKFLOW workflow
+ 
+ * Running WORKFLOW
+ 
+ === Description here ===.
```

Comment:
This is just a placeholder, no description. Shall we add a diagram here? Or since it is a one-step workflow it is not worth it?

Reply:
updated